### ISSUE TITLE
Incorrect code generated using template literals

### ISSUE THREAD

**Issue by Jim-Bar** [coderaiser/minify#120]
Hello,

We stumbled upon what seems to be a bug in Minify using template literals. Consider the following code:

```js
const a = 'a';
console.log(`${a}`);
let b = 'b1';
b = 'b2';
const c = `${b}`;
console.log(`${c}`);
```

Running it without minifying yields (as expected):

```
a
b2
```

Minifying it outputs the following code:

```
var a='a',c=`${b}`;console.log(`${a}`);let b='b1';b='b2';console.log(`${c}`);
```

This code has an issue: `b` is used before it is defined. Indeed, running it I get an error:

```
var a='a',c=`${b}`;console.log(`${a}`);let b='b1';b='b2';console.log(`${c}`);
            ^

ReferenceError: Cannot access 'b' before initialization
```

Now if I remove `a` and its console log (which has no relation with the rest), or if I replace at least one template literal by the variable itself, Minify generates working code.

Of course this code is trivial, but we reproduce the problem in a more concrete case using `URLSearchParams` for formatting URL: our query parameters get ignored.

**Comment by coderaiser** [comment]
Looks like it is related to [`minify/mungle-names`](https://github.com/coderaiser/putout/tree/master/packages/plugin-minify#merge-variables).

**Comment by coderaiser** [comment]
Just fixed in [`@putout/minify v3.8.1`](https://github.com/putoutjs/minify/releases/tag/v3.8.1) 🎉. Is it works for you?
